### ISSUE TITLE
App hangs with infinite progress "Your archive is being saved. Exiting automatically" ?

### ISSUE THREAD

**Issue by esutton** [buttercup/buttercup-desktop#902]
Version 1.18.0 (1.18.0)
macOS 10.15.3

The latest update is causing this apparently:

![Screen Shot 2020-04-15 at 9 46 54 AM](https://user-images.githubusercontent.com/6115047/79351274-1954c400-7efe-11ea-8974-6b14d238d692.png)

Why so many update releases?  

Practically everyday Buttercup pauses at start-up for what I assume is checking for and installing updates?  This makes me Worry for the stability of a password keeper.  Would it be better to batch them up and release them at a slower rate?

After looking at your not very frequent releases, I do not understand what is going on when I start Buttercup?
- https://github.com/buttercup/buttercup-desktop/releases

**Comment by esutton** [comment]
Fixed:
1) I closed Buttercup.  
2) Killed a hidden buttercup I found in task manager
3) Deleted Buttercup
4) Downloaded and re-installed

**Comment by crowell-snipes** [comment]
Same here, but I have yet to delete/reinstall.

**Comment by crowell-snipes** [comment]
I have this error in console - 
```
/Applications/Buttercup.app/Contents/Resources/app.asar/dist/main.css:1 
Failed to load resource: net::ERR_FILE_NOT_FOUND
```

**Comment by willisplummer** [comment]
fixed when i installed the latest version (1.20.4)

**Comment by perry-mitchell** [comment]
Stale
